### ISSUE TITLE
Visual Studio 2022 Error Compiling

### ISSUE THREAD

**Issue by Ballistiks** [php/php-sdk-binary-tools#14]
![image](https://github.com/php/php-sdk-binary-tools/assets/105979162/5049d5c1-2f7f-47c5-b4ac-517f5ab771c9)

On phpsdk_deps --update give the error above. I've using visual studio 2022.

**Comment by jay94ks** [comment]
This can be skipped by editing that `Config.php` file to:
```
............................
			/* Evaluate CRTs, to avoid ambiquity. */
			$crt = key($branches[$current_branch_name]);
			$data = $branches[$current_branch_name][$crt];
			if ($crt != $cur_crt) {
				// throw new Exception("The passed CRT '$cur_crt', doesn't match any available for branch '$current_branch_name'");
			}
		}

....
```

This will prepare dependency packages for `vs16`, but everything works fine on my test build.

```
phpsdk_deps --update --branch master

.\buildconf.bat
.\configure.bat --!ZTS_STATE!-zts --disable-cgi --enable-embed=shared --enable-cli-win32

nmake
```

**Comment by cmb69** [comment]
This issue is basically resolved as of 19c8ccbf072370b0ed99bd8734ed09926549d16f, and for roughly a week, vs17 packages are available from downloads.php.net. So if you're still using `php-sdk-2.2.0`, switch to `master` or wait for a while until `php-sdk-2.3.0` is available.
